Bottom: 526d6d03711a7d573e860c83c136c48098fba323
Top:    4a1b07e0481a064e9839ebc6c2edb2273bca5aaf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-06-12 20:01:48 +0100

Refresh of improve-auto-enforce-drc

---

diff --git a/src/line.c b/src/line.c
index 745c854..5f6898b 100644
--- a/src/line.c
+++ b/src/line.c
@@ -500,7 +500,7 @@ drc_line (PointType *end)
  * adjusts the end point until there is no intersection or
  * it winds up back at the start. If way is false it checks
  * straight start, 45 end lines, otherwise it checks 45 start,
- * straight end. 
+ * straight end.
  *
  * It returns the straight-line length of the best answer, and
  * changes the position of the input point to the best answer.
@@ -658,7 +658,7 @@ drc_lines2 (PointType *end, bool way)
  * adjusts the end point until there is no intersection or
  * it winds up back at the start. If way is false it checks
  * straight start, 45 end lines, otherwise it checks 45 start,
- * straight end. 
+ * straight end.
  *
  * It returns the straight-line length of the best answer, and
  * changes the position of the input point to the best answer.
